include/mbedtls/md.h

```diff
@@ -63,65 +65,69 @@ typedef enum {
 #endif
 
 /**
- * Opaque struct defined in md_internal.h
+ * Opaque struct defined in md_internal.h.
  */
 typedef struct mbedtls_md_info_t mbedtls_md_info_t;
 
 /**
- * Generic message digest context.
+ * The generic message-digest context.
  */
 typedef struct {
-    /** Information about the associated message digest */
+    /** Information about the associated message digest. */
```

Comment:
Style: There is a hyphen missing here.

Reply:
No. See my previous explanation re. when to hyphenate it and when not.